osm2streets/src/lanes/mod.rs

```diff
@@ -87,6 +88,47 @@ impl LaneType {
         }
     }
 
+    /// Determines if the lane is a travel lane that is represented in OSM `*:lanes` tags.
+    /// Note that the `lanes` tag counts car driving lanes, excluding bike lanes, whereas the
+    /// `:lanes` suffix specifies that each lane, including bike lanes, should have a value between
+    /// `|`s. This function identifies the latter kind.
+    pub fn is_tagged_by_lanes_suffix(&self) -> bool {
+        match self {
+            LaneType::Driving => true,
+            LaneType::Biking => true, // FIXME depends on lane vs track
+            LaneType::Bus => true,
+            LaneType::Parking => false,
+            LaneType::Sidewalk => false,
+            LaneType::Shoulder => false,
+            LaneType::SharedLeftTurn => true,
+            LaneType::Construction => true,
+            LaneType::LightRail => false,
+            LaneType::Buffer(_) => false,
+            LaneType::Footway => false,
+            LaneType::SharedUse => false,
+        }
+    }
+
+    /// Determines if the lane is part of the roadway, the contiguous sealed surface that OSM
+    /// mappers consider the "road".
+    pub fn is_roadway(&self) -> bool {
```

Comment:
This function (and `is_tagged_by_lanes_suffix` above) might need to be moved to `LaneSpec`, with some extra info stored to answer them properly. Or this gets clearer when we improve `LaneType` to tease apart the multiple different properties it is trying to represent.

The `is_roadway` concept that I use here might need to be a field on `LaneSpec` that is determined at parsing time.

Reply:
Adding to `LaneSpec` and/or refining `LaneType` sounds fine to me. I still would like to avoid the full generality of `osm2lanes` though.. https://github.com/a-b-street/osm2lanes/blob/main/osm2lanes/src/road/lane.rs can express nonsensical things like a parking lane for `Foot`.

Rephrasing... if we start to refactor `LaneType` dramatically, let's please do it in a separate PR and a bit slowly, so I can keep downstream A/B Street code abreast of changes. #91 is an example that would probably be great to do in this repo, but I'd like time to support it everywhere

Reply:
> separate PRs and a bit slowly.

Absolutely, I'm enjoying the standard set by supporting A/B Street at each iteration.

> I still would like to avoid the full generality of osm2lanes though.

This is an opportunity to see what a useful osm2lanes API would look like, so we can carve osm2lanes down into the support for this lib. `RoadPosition`/`Placement` is an example of the kind of complexity we end up needing (i think). 

I think it's ok that the types (aka vocabulary) let you say semantically nonsense things like "pedestrian parking". What if someone tagged it as park of crowd/queue management at some arena or something? We shouldn't crash. Parsing tags into fields seems simpler than trying to classify into an enum of distinct types right at the beginning. If parking can be tagged with access restrictions, then it's up to the renderer to scratch its head when it tries to find lane markings for foot parking.